src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/Touch/TouchBehavior.Methods.shared.cs

```diff
@@ -0,0 +1,168 @@
+﻿namespace CommunityToolkit.Maui.Behaviors;
+
+public partial class TouchBehavior : IDisposable
+{
+	readonly NullReferenceException nre = new(nameof(Element));
```

Comment:
I believe this can be `static`, that way we have just one instance for all TouchBehavior used by devs.
```suggestion
	static readonly NullReferenceException nre = new(nameof(Element));
```

Reply:
Updated